inginious/agent/docker_agent/_docker_interface.py

```diff
@@ -21,6 +21,14 @@ class DockerInterface(object):  # pragma: no cover
         Docker-py itself is already well tested.
     """
 
+    def __init__(self, type, runtime):
+        """
+        :param type: type of the container
+        :param runtime: runtime used by docker
+        """
+        self.type = type
+        self.runtime = runtime
```

Comment:
You should check the runtime. As its given directly to docker, it would be beneficial to avoid having possible injections here.